src/analysis/retail/evoker/preservation/modules/talents/RenewingBreath.tsx

```diff
@@ -0,0 +1,58 @@
+import { formatNumber } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import { TALENTS_EVOKER } from 'common/TALENTS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import { calculateEffectiveHealing, calculateOverhealing } from 'parser/core/EventCalculateLib';
+import Events, { HealEvent } from 'parser/core/Events';
+import ItemHealingDone from 'parser/ui/ItemHealingDone';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import TalentSpellText from 'parser/ui/TalentSpellText';
+
+const INCREASE_PER_POINT = 0.15;
```

Comment:
We either need to put all these types of values in constants or just move the ones in constants into their respective modules. Either way, but lets keep it consistent